src/common/models/Projections.ts

```diff
@@ -95,6 +95,17 @@ export class Projections {
     };
   }
 
+  hasMetric(metric: Metric): boolean {
+    if (metric === Metric.FUTURE_PROJECTIONS) {
+      return (
+        this.baseline?.hasHospitalProjections &&
+        this.projected.hasHospitalProjections
```

Comment:
Should the metric simply return null for this case?  this logic almost feels like it could live on the metric: `metric.hasValue()`.  Note that I'm suggesting this without much context :) just a first impression

Reply:
`Metric` is just an enum, so I'm not sure what you mean by `metric.hasValue()`.  As for returning null, `Metric.FUTURE_PROJECTIONS` is weird in that it has a metric chart, but no metric value (we don't choose a specific value and grade it)...  so it ends up special-cased in places like this.  It's not ideal, but I don't have a better solution offhand.